vtm/src/org/oscim/layers/tile/buildings/S3DBUtils.java

```diff
@@ -1251,15 +1251,14 @@ private static Integer getIndexStart(List<Byte> simpleAngles, List<Float> length
 
     /**
      * @param material the material as string (see http://wiki.openstreetmap.org/wiki/Key:material and following pages)
-     * @param roof     declare if material is used for roofs
      * @return the color as integer (8 bit each a, r, g, b)
      */
-    public static int getMaterialColor(String material, boolean roof) {
+    public static int getMaterialColor(String material) {
 
-        if (roof) {
-            if ("glass".equals(material))
-                return Color.fade(Color.get(130, 224, 255), 0.9f);
+        if (material.charAt(0) == '#') {
+            return Color.parseColor(material, Color.CYAN);
```

Comment:
By the way why cyan as fallback color?

Reply:
No idea why. Adopted it from `getColor`. Can use any value you want :)